erpnext/accounts/doctype/financial_report_template/financial_report_engine.py

```
@@ -0,0 +1,1686 @@
+
+			# Execute
+			self.collect_financial_data(context)
+			self.process_calculations(context)
+			self.format_report_data(context)
+			self.apply_view_transformation(context)
+
+			# Chart
+			self.generate_chart_data(context)
+			return context.get_result()
+
+		except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)